graphql-async/test/jbuild

```diff
@@ -0,0 +1,10 @@
+(jbuild_version 1)
+
+(executables
+ ((libraries (graphql_async alcotest async))
```

Comment:
It can be Async_kernel I ran into some problems wrt 0.9.0 and 113. Could you upgrade the lib to use the newest async first and then I'll sort this out? Or should I just include the async upgrade in this PR?